production.yml

```diff
@@ -32,7 +34,22 @@ services:
     volumes:
       - ${UPLOADED_DATA_DIRECTORY:-./data}:/data
     command: /start
+    labels:
+      traefik.enable: "true"
+      # Incoming http to our domain name via websecure entrypoint gets routed to django, modulo priority
+      traefik.http.routers.djangoroute.rule: "Host(`${TRAEFIK_ROUTER_DOMAIN_NAME:?traefik_router_domain_name_missing}`)"
+      traefik.http.routers.djangoroute.entrypoints: "websecure"
+      traefik.http.routers.djangoroute.tls.certresolver: "letsencrypt"
+      traefik.http.routers.djangoroute.priority: 10
+      # Django internally listens on port 5000 by default - presumably we could configure django to listen on port 80 or 443,
+      # but we can also just tell traefik to listen here.
+      traefik.http.services.djangoroute.loadbalancer.server.port: 5000
+      # Define the "csrf" middleware and make djangoroute use it
+      traefik.http.middlewares.csrf.headers.hostsProxyHeaders: "X-CSRFToken"
```

Comment:
Maybe this line should ve in traefik section

Reply:
It works this way according to the API (shows middleware successfully) but it does not work if I delete it from both django and nginx and put it in traefik labels. So apparently it needs to be duplicated locally for each container that uses it.